src/types/SpringDataRestResponse.tsx

```diff
@@ -0,0 +1,12 @@
+interface SpringDataRestResponse<T> {
+  _embedded: {
+    works: T[];
```

Comment:
This key needs to be not tied to the specific resource name for the type to be reused